### PR TITLE
🔀🌐[it_IT] add missing translations

### PR DESCRIPTION
**Pull request by sup3r93**
Update:

- Addition of new buildings (I was unable to insert the extensions of all the buildings due to lack of information)
- Addition of new vehicles
- Editing of the translation files of the first modules.

**What kind of update does this PR provide?** *Please check*

- [x] new translations / updated translations / translation fixes
- [ ] a new feature
- [ ] a new module
- [ ] a bugfix for an existing feature / module
- [ ] improvement of style or user experience
- [ ] other: *Please fill out*

**Which language(s) did you add/update translations for?**
*use the xx_XX notation for exact language!*
* it_IT

**What is included in your update?**
In each file that I modified, I inserted in the commit the relevant changes made

**Additional notes**
Please add any further notes here!

### CODE REVIEW

File: src/i18n/it_IT/vehicles.ts [hunk 1]
```diff
@@ -606,4 +606,310 @@ export default {
         icon: 'car-side',
         possibleBuildings: [6, 19],
     },
+    42: {
+        caption: 'Auto della polizia locale',
+        color: '#0a2bff',
+        credits: -1,
```
Comment (Cr4zyc4k3) [inline src/i18n/it_IT/vehicles.ts:612]: -1 doesnt seem correct

**Review by Cr4zyc4k3 — CHANGES_REQUESTED** [review]

File: src/i18n/it_IT/vehicles.ts (hunk 1 above)
Comment (sup3r93) [reply src/i18n/it_IT/vehicles.ts]: > -1 doesnt seem correct

I managed to add the credits and coins for the purchase of the vehicle:
42 - 43 - 44 - 45
For the others from 46 onwards, I don't have the suitable buildings, and I can't verify how much they cost

File: src/i18n/it_IT/buildings.ts [hunk 2]
```diff
@@ -823,4 +823,136 @@ export default {
...
+        },
+        extensions: [],
+        levelcost: [],
+        maxBuildings: 'Nessun limite',
+        maxLevel: 11,
+        special: '',
+        startPersonnel: 6,
+        startVehicles: [],
+        schoolingTypes: ['Accademia Navale'],
+        schools: [8],
```
Comment (jxn-30) [inline src/i18n/it_IT/buildings.ts:865]: 27, as ID of accademia navale is 27

File: src/i18n/it_IT/buildings.ts [hunk 3]
```diff
@@ -823,4 +823,136 @@ export default {
...
+        maxLevel: 0,
+        special: '',
+        startPersonnel: 6,
+        startVehicles: [],
+        schoolingTypes: ['Polizia'],
+        schools: [8],
+        startParkingLots: 1,
+        startCells: 0,
+        icon: 'building-shield',
+    },
```
Comment (jxn-30) [inline src/i18n/it_IT/buildings.ts:891]: that doesn't seem right, it should be a school, similar to buildings 1, 3 and 8.

File: src/i18n/it_IT/vehicles.ts [hunk 4]
```diff
@@ -606,4 +606,310 @@ export default {
...
+    },
+    75: {
+        caption: 'Autobotte AIB PC',
+        color: '#0a2bff',
+        credits: -1,
+        coins: -1,
+        staff: { min: 1, max: 2 },
+        icon: 'car-side',
+        possibleBuildings: [30],
+    },
```
Comment (jxn-30) [inline src/i18n/it_IT/vehicles.ts:914]: trainings for staff missing here probably, but they can also be added later in theory.

**Review by jxn-30 — CHANGES_REQUESTED** [review]

File: src/i18n/it_IT/vehicles.ts (hunk 4 above)
Comment (sup3r93) [reply src/i18n/it_IT/vehicles.ts:914]: I know but at this moment I don't know what the training programs are for the aforementioned vehicles

File: src/i18n/it_IT/buildings.ts (hunk 2 above)
Comment (sup3r93) [reply src/i18n/it_IT/buildings.ts]: Solved

File: src/i18n/it_IT/buildings.ts (hunk 3 above)
Comment (sup3r93) [reply src/i18n/it_IT/buildings.ts]: Solved

**Review by jxn-30 — APPROVED** [review]

**Review by Cr4zyc4k3 — COMMENTED** [review]
Execept the missing vehicle prices it looks good to me

**Review by Cr4zyc4k3 — APPROVED** [review]

**Pull request merged**